html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,14 @@
 <?php
-for ($i = 1; $i <= 100; $i++) {
+for($i=1; $i<=100; $i++) {
```

Comment:
レビューNGにするほどではないですが演算子の左右の間隔をあけた方が見やすいですね！せっかくなので治しましょう
```suggestion
for ($i = 1; $i <= 100; $i++) {
```